tests/test_persistence.py

```diff
@@ -709,7 +590,7 @@ def test_trade_close(limit_buy_order_usdt, limit_sell_order_usdt, fee):
     assert trade.close_date == new_date
 
 
-@pytest.mark.usefixtures("init_persistence")
+@ pytest.mark.usefixtures("init_persistence")
```

Comment:
I'd not do that (and i'm not sure which stylesheet you've found this).

It looks odd - and is pretty dangerous when trying to find things - as it derives from the general style we're using elsewhere for decorators.

Reply:
Sorry, it's some auto-formatting thing that only happens sometimes, and  I haven't figured out what makes it happen

Reply:
It's pretty strange, too - as it's not applied consistently (there's a ton of `@pytest` occurances in this same file which are not formatted this way).